src/v2/Apps/Auctions/Components/AuctionArtworksRail/__tests__/AuctionArtworksRail.jest.tsx

```diff
@@ -45,4 +61,17 @@ describe.skip("AuctionArtworksRail", () => {
     expect(text).toContain("Test Href")
     expect(text).toContain("Ends Apr 10 at 8:27pm UTC")
   })
+
+  it("tracks clicks", () => {
+    const wrapper = getWrapper()
+    wrapper.find("RouterLink").first().simulate("click")
+    expect(trackEvent).toHaveBeenCalledWith({
+      action: "clickedArtworkGroup",
+      context_module: "currentAuctions",
+      destination_page_owner_id: '<mock-value-for-field-"internalID">',
+      destination_page_owner_slug: '<mock-value-for-field-"slug">',
+      destination_page_owner_type: "sale",
+      type: "viewAll",
+    })
+  })
```

Comment:
@artsy/grow-devs - Not sure if y'all are familiar with this strategy, but once the in general tracking setup is added to the test file (mocks etc), and tracking added to the file under test, its easy to find  the values here by leaning on jest's diffing function: 

Steps
1) 
```tsx
expect(trackEvent).toHaveBeenCalledWith('hey now') 
```
2) Run test, check jest's diff
```
 Expected: "hey now"
    Received: {"action": "clickedArtworkGroup", "context_module": "worksByArtistsYouFollowRail", "context_page_owner_id": undefined, "context_page_owner_slug": undefined, "context_page_owner_type": undefined, "destination_page_owner_id": "<mock-value-for-field-\"internalID\">", "destination_page_owner_slug": "<mock-value-for-field-\"slug\">", "destination_page_owner_type": "artwork", "horizontal_slide_position": 0, "type": "thumbnail"}
```
3) Copy that received section into test and delete the props that are undefined:
```tsx
  expect(trackEvent).toHaveBeenCalledWith({
      action: "clickedArtworkGroup",
      context_module: "currentAuctions",
      destination_page_owner_id: '<mock-value-for-field-"internalID">',
      destination_page_owner_slug: '<mock-value-for-field-"slug">',
      destination_page_owner_type: "sale",
      type: "viewAll",
  })
    ```
4) Verify that it looks right by comparing against the file being tested. 

Note that this jest-leaning can be done with all kinds of things.